src/sentry/static/sentry/app/views/organizationIntegrations/addIntegrationButton.jsx

```diff
@@ -48,12 +50,16 @@ export default class AddIntegrationButton extends React.Component {
 
   handleAddIntegration = provider => {
     const name = 'sentryAddIntegration';
+    debugger;
```

Comment:
oops

Reply:
how did this pass precommit hooks :<

Reply:
https://travis-ci.org/getsentry/sentry/jobs/397900763#L655 It actually only shows up as a warning